tnz/zti.py

```diff
@@ -115,6 +116,11 @@ def __init__(self, stdin=None, stdout=None):
         if os.getenv("ZTI_AUTOSIZE"):
             self.autosize = True
 
+        self.macros_dir = os.getenv("ZTI_MACROS_DIR")
+        if self.macros_dir is None or not os.path.isdir(
+           self.macros_dir):
```

Comment:
Don't hide the error if the user has `ZTI_MACROS_DIR=not_a_dir`. I suggest just removing that part of the check.